CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [v1.1.2] - 2018-03-14
```

Comment:
Add a link at the bottom (see line 71)

Reply:
Done https://github.com/cumulus-nasa/cumulus/pull/244/commits/98c53647f9c574114dfcbd28fb6ec27f09c4e4b7